src/components/code_snippets/_bitbucket-semgrep-app-sast.mdx

```diff
@@ -2,31 +2,31 @@
 image: atlassian/default-image:latest
 
 pipelines:
-  default:
-    - parallel:
+  branches:
+    main:
+    - step:
+        image: semgrep/semgrep
+        name: Semgrep Scan on Push
+        script:
+          - semgrep ci
+
+  pull-requests:
+    '**': # This applies to pull requests for all branches
       - step:
-          name: 'Run Semgrep scan with current branch'
-          deployment: dev # https://support.atlassian.com/bitbucket-cloud/docs/set-up-and-monitor-deployments/
           image: semgrep/semgrep
+          name: Semgrep Scan on PR
           script:
-            # The following variables are required to set up a scan connected to Semgrep Cloud Platform:
-            - export SEMGREP_APP_TOKEN=$SEMGREP_APP_TOKEN
```

Comment:
I don't think we ought to remove this line :) Can you restore it?

Reply:
Yeah, good point - we may need to not only restore this but make sure it's in all the right places.

Reply:
@s-santillan SEMGREP_APP_TOKEN should still be in there. That is just my mistake.

Reply:
Added it into the 3 events. Resolving.